src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -3120,6 +3120,7 @@ public class LocalReadingPosition
     public string PlatformOffset { get; set; }
 }
 
+#if !uapaot
 public class NativeDllWrapper
```

Comment:
Just wondering since I didn't see the PR that added this, but did we add any other PInvoke to NativeDll.dll or is this the only one?